FAQ.md

```diff
@@ -327,20 +365,72 @@ You can scale TiDB as your business grows.
 
 ### Monitor
 
-Should I deploy the TiDB monitoring framework (Prometheus + Grafana) on a standalone machine or on multiple machines? What is the recommended CPU and memory?
+#### Should I deploy the TiDB monitoring framework (Prometheus + Grafana) on a standalone machine or on multiple machines? What is the recommended CPU and memory?
 
 The monitoring machine is recommended to use standalone deployment. It is recommended to use 8 core CPU, 32 GB+ memory and 500 GB+ hard disk.  
 
 #### The monitor can't show all the metrics.
 
 Check the time difference between the machine time of the monitor and the time within the cluster. If it is large, you can correct the time and the monitor will display all the metrics.
 
+#### How to configure to monitor Syncer status?
+
+Download [Syncer Json](https://github.com/pingcap/docs/blob/master/etc/Syncer.json) to Grafana, modify the Prometheus configuration file, add the following:
```

Comment:
Download -> Download and import

Reply:
modify the Prometheus configuration file, add the following:
-> edit the Prometheus configuration file and add the following content: